Gui/opensim/view/src/org/opensim/view/Installer.java

```diff
@@ -77,7 +80,21 @@ public boolean closing() {
 
     public void restored() {
         super.restored();
-        System.setProperty ("netbeans.buildnumber", "4.0.Beta"); // Should get that from JNI but sometimes doesn't work'
+        // The following block reads the file build_id.properties created during build/install
+        // and recovers netbeans.buildnumber to use it as a System property.
+        // Fomrat is YYYY-MM-DD timeInUTC abbreviated-commit-hash, will hsow on AboutJPanel 
+        Properties prop = new Properties();
+        InputStream input = null;
+        try {
+            input = new FileInputStream("build_id.properties");
```

Comment:
Good idea @chrisdembia to use the .conf file rather than figure out cross platform location (I now remember creating a function to return platform specific install dir), I think we'll have to use -J-Dnetbeans.buildnumber=@BUILD_ID@ then since this property is what's queried by the About dialog.